examples/tests/sdk_tests/api/delete.w

```diff
@@ -0,0 +1,38 @@
+
+bring cloud;
+bring http;
+
+let http_DELETE = http.HttpMethod.DELETE;
+let api_DELETE = cloud.HttpMethod.DELETE;
+
+let api = new cloud.Api();
+
+api.delete("/path", inflight (req: cloud.ApiRequest): cloud.ApiResponse => {
+    assert(req.method == api_DELETE);
+    assert(req.query?.get("all") == "true");
+    assert(req.query?.get("page") == "6");
+    assert(req.path == "/path");
+
+    return cloud.ApiResponse {
+        status: 200,
+        body: req.query?.get("page")
+    };
```

Comment:
Should be indented 2 spaces for consistency with our other examples (someday we will have a Wing formatter 🤞)